pkgdb2/ui/packages.py

```diff
@@ -905,6 +920,7 @@ def package_unretire(namespace, package, full=True):
                         flask.flash(str(err), 'error')
                         SESSION.rollback()
                     except SQLAlchemyError, err:
+                        print err
```

Comment:
Is this a leftover from debugging? Perhaps we should instead use the logging functions.